src/components/about-us/AboutHero.tsx

```diff
@@ -1,13 +1,62 @@
-import { YouTubeEmbed } from "@/components/youtube-embed/YoutubeEmbed";
 import { FC } from "react";
+import Image from "next/image";
+import { Callout } from "@radix-ui/themes";
 
 const AboutHero: FC = () => (
-  <div className="py-8 md:py-20 mx-auto" style={{ maxWidth: 800 }}>
-    <YouTubeEmbed
-      title="About Distribute Aid!"
-      url="https://www.youtube.com/embed/msizPweg3kE"
-    />
-  </div>
+  <>
+    <div className="absolute inset-0 bg-navy-900 -z-10 h-[609px]"></div>
```

Comment:
**suggestion (non-blocking):** Suggestion (non-blocking): Would be good to use RadixUI's [Box Component](https://www.radix-ui.com/themes/docs/components/box) if possible.

**suggestion (blocking):** Instead of using an absolutely positioned element for the background, can we wrap the hero in an element with the background color set + some bottom margin, and then relatively position the content downwards?

**suggestion (non-blocking):** Use RadixUI's [Theme component](https://www.radix-ui.com/themes/docs/theme/overview) to set background colors if possible.